src/Model/MusicManager.php

```diff
@@ -19,19 +20,50 @@ public function selectAllList()
         return $this->pdo->query($query)->fetchAll();
     }
 
-    public const JOINED_TABLE = 'musical_genre';
+    public function selectAllItems()
+    {
+        $query = "SELECT music.id, music.title, music.author, music.source,
+        music.music_image, music.musical_genre_id, genre.genre_name
+            FROM music INNER JOIN musical_genre as genre
```

Comment:
It's a good idea to add the const JOINED_TABLE but it's useless if you don't use it in your queries (Same here with TABLE by the way)